config/environments/production.rb

```diff
@@ -33,6 +33,9 @@
   # Store uploaded files on the local file system (see config/storage.yml for options).
   config.active_storage.service = :local
 
+  # Store files on Amazon S3.
+  config.active_storage.service = :amazon
+
```

Comment:
Please check this config because it is duplicated in the file and is not required in the ticket.